h/crossmemory.h

```diff
@@ -848,6 +850,12 @@ CrossMemoryServerName cmsMakeServerName(const char *nameNullTerm);
 #define CMS_LOG_SRVC_ENTRY_OCCUPIED_MSG_TEXT    "Service entry %d is occupied"
 #define CMS_LOG_SRVC_ENTRY_OCCUPIED_MSG         CMS_LOG_SRVC_ENTRY_OCCUPIED_MSG_ID" "CMS_LOG_SRVC_ENTRY_OCCUPIED_MSG_TEXT
 
+#ifndef CMS_LOG_CLEAN_LPA_MSG_ID
+#define CMS_LOG_CLEAN_LPA_MSG_ID                CMS_MSG_PRFX"0247W"
+#endif
+#define CMS_LOG_CLEAN_LPA_MSG_TEXT              "Unsafe LPA development mode is enabled"
```

Comment:
What exactly is unsafe? Are we using some kind of brute force method that may result in system integrity problems?

Reply:
The messages has been changed. Please take a look.